README.md

```diff
@@ -1,127 +1,220 @@
+<h3 align="center">Unity SDK</h3>
 
-## 1. Install SDK
+  <p align="center">
+The Ankr Unity SDK provides an easy way to interact with Web3 and to work with contracts deployed on the blockchain. As a plugin it easily integrates with MetaMask on Android and iOS.
+    <br />
+    <a href="https://github.com/Ankr-network/game-unity-demo">View Demo</a>
+  </p>
+</div>
 
-1) Download  mirageSDK.unitypackage from [latest release](https://github.com/Ankr-network/unity-web3/releases).
-2) Drag and drop it to the Assets folder.
-3) Import all 
+<!-- TABLE OF CONTENTS -->
+<details>
+  <summary>Table of Contents</summary>
+  <ol>
+    <li>
+      <a href="#install-sdk">Install SDK</a>
+    </li>
+	<li>
+      <a href="#what's-in-the-sdk">What's in the SDK</a>
+    </li>
+    <li><a href="#getting-started">Getting Started</a></li>
+	<ul>
+        <li><a href="#prequisites">Prerequisites</a></li>
+        <li><a href="#use-cases">Use Cases</a></li>
+	</ul>
+    <li>
+      <a href="#01-connect-web3-wallet">Connect Web3 Wallet</a></li>
+	  <a href="#02-perform-updates-to-nfts">Perform Updates to NFTs</a>
+      <ul>
+        <li><a href="#signing-transactions">Signing Transactions</a></li>
+        <li><a href="#sending-transactions">Sending Transactions</a></li>
+        <li><a href="#getdata-method">GetData Method</a></li>
+        <li><a href="#callmethod">CallMethod</a></li>
+      </ul>    
+    <li><a href="#current-erc-proposals">Current ERC Proposals</a></li>
+	<li><a href="#view-full-examples">View Full Examples</a></li>
+  </ol>
+</details>
 
-## 2. Getting started
 
-To start work with plugin you need to acquire our SDK wrapper instance through one of .GetSDkInstance methods.
-```c#
-var mirageSDKWrapper = MirageSDKWrapper.GetSDKInstance(ERC20ContractInformation.ProviderURL);
-```
+<!-- Installation -->
 
-To work with contracts you need to get a IContract instance from our SDKWrapper, for that call GetContract(...)
+##  🏗 Install SDK
 
-```c#
-string abi = "...";
-string contract_address = "0x...";
+1. Download `mirageSDK.unitypackage` from the latest release.
 
-IContract contract = mirageSDKWrapper.GetContract(contract_address, abi);
-```
+2. In your project, locate the 'Assets' folder. Move `mirageSDK.unitypackage` into this folder.
+
+3. 'Import all' to have access to all SDK capabilities.
+
+## 👀 What's in the SDK
+
+The SDK is designed to make it super easy to get started with Game development by enabling connection and interaction across different blockchains. 
+
+ * Contains a huge range of classes, plugins and example scripts for a variety of use cases. 
+
+* Nethereum libraries provide support for web requests using RPC over Http. 
+
+* Ankr RPC network infrastructure provides fast and easy connection to multiple chains. 
+
+
+## Getting started
+
+### 🧰 Prerequisites
+
+1. Smart Contracts must already be deployed on the blockchain. You have the Smart Contract addresses and ABI
+2. Web3.js or Ethers.js
```

Comment:
Why is it a prerequisites? We are trying to provide a full functionality of web3.js/ethers.js, but it is not required to start using our sdk